### PR TITLE
readme: improvements

### PR DESCRIPTION
**Pull request by illwieckz**
improve a bit the readme

everything stating with “Launch Instructions” is left untouched
as it would be rewritten after some scripts being improved

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,25 @@ This repository contains the gamelogic of the game Unvanquished.
 You need to download the game's assets in addition to that to make it run.
 See below for build and launch instructions.
 
+## Workspace requirements
+
+To fetch and build Unvanquished, you'll need:
+`git`,
+`cmake`,
+`python` ≥ 2,
```
Comment (slipher) [inline README.md]: Python == 2 is required if you want to build NaCl. Otherwise LGTM.

Comment (illwieckz) [reply README.md]: is building NaCl required to build Unvanquished? I guess the python requisite is for some code generation unrelated to NaCl, right?

Comment (slipher) [reply README.md]: To build NaCl VMs. The 2 or 3 requirement is for CBSE (sgame).

Comment (illwieckz) [reply README.md]: ah, that's not cool… then it's probably better to just say python2 if NaCl requires it and CBSE can deal with it…

**Comment by illwieckz** [comment]
edited the irc badge to use generic badge syntax since shield.io just dropped irc template.

**Comment by slipher** [comment]
LGTM

**Pull request merged**